index.bs

```diff
@@ -417,22 +413,17 @@ quantization algorithm=]:
 
 Automation {#automation}
 ==========
-This section extends the [=automation=] section defined in the Generic Sensor API [[GENERIC-SENSOR]]
-to provide mocking information about the ambient light levels for the purposes of testing a user agent's
-implementation of [=Ambient Light Sensor=].
+This section extends [[GENERIC-SENSOR#automation]] by providing [=Ambient Light Sensor=]-specific virtual sensor metadata.
```

Comment:
Can you add an extra empty line before this paragraph? Apparently Bikeshed requires that in order to wrap all the paragraphs around `<p>` tags, which is something I've just noticed and will also need to fix in the Gyroscope spec.

Reply:
Empty line added in https://github.com/w3c/ambient-light/pull/87/commits/7bbbbfad1940bd66a2bdf3fa201a4698f511da89

Now algorithm paragraph also better shows as its own separate paragraph.